tools/packer/repo/config.toml

```diff
@@ -0,0 +1,36 @@
+[API]
```

Comment:
The reason this file was added rather than sticking to the default config is so I could enable the UFW firewall with predictable libp2p port numbers.